src/components/SampleGraphs/sampleGraphs.js

```diff
@@ -1,44 +1,47 @@
-        authorLink: "https://networkofthrones.wordpress.com/",
-        description: `A dataset containing interactions between the characters across the first 7 seasons of the popular TV show.`,
-        queries: [
-            `CREATE CONSTRAINT ON (c:Character) ASSERT c.id IS UNIQUE`,
-            `UNWIND range(1,7) AS season
+      name: "Game of Thrones",
+      author: "Andrew Beveridge",
+      authorLink: "https://networkofthrones.wordpress.com/",
+      description: `A dataset containing interactions between the characters across the first 7 seasons of the popular TV show.`,
+      queries: [
+        `CREATE CONSTRAINT ON (c:Character) ASSERT c.id IS UNIQUE`,
+        `UNWIND range(1,7) AS season
 LOAD CSV WITH HEADERS FROM "https://github.com/neo4j-apps/neuler/raw/master/sample-data/got/got-s" + season + "-nodes.csv" AS row
 MERGE (c:Character {id: row.Id})
 ON CREATE SET c.name = row.Label`,
-            `UNWIND range(1,7) AS season
+        `UNWIND range(1,7) AS season
 LOAD CSV WITH HEADERS FROM "https://github.com/neo4j-apps/neuler/raw/master/sample-data/got/got-s" + season + "-edges.csv" AS row
 MATCH (source:Character {id: row.Source})
 MATCH (target:Character {id: row.Target})
 CALL apoc.merge.relationship(source, "INTERACTS_SEASON" + season, {}, {}, target) YIELD rel
-SET rel.weight = toInteger(row.Weight)`
-        ],
-        algorithms: [
-            {
-                name: "Page Rank",
-                category: "Centralities",
-                description: "Find the most influential characters in Westeros and elsewhere."
-            },
-            {
-                name: "Shortest Path",
-                category: "Path Finding",
-                description: "Who can introduce Jon to Daenerys? Or Sansa to Drogo?"
-            }
-        ]
+SET rel.weight = toInteger(row.Weight)`,
+      ],
+      algorithms: [
+        {
+          name: "Page Rank",
+          category: "Centralities",
+          description:
+            "Find the most influential characters in Westeros and elsewhere.",
+        },
+        {
+          name: "Shortest Path",
+          category: "Path Finding",
+          description: "Who can introduce Jon to Daenerys? Or Sansa to Drogo?",
+        },
+      ],
     },
 
     "European Roads": {
-        name: "European Roads",
-        author: "Lasse Westh-Nielsen",
-        authorLink: "http://lassewesth.blogspot.com/2018/07/the-international-e-road-network-and.html",
-        description: `A dataset containing European Roads.`,
-        queries: [
-            `CREATE CONSTRAINT ON (p:Place) ASSERT p.name IS UNIQUE`,
-            `USING PERIODIC COMMIT 1000
+      name: "European Roads",
+      author: "Lasse Westh-Nielsen",
+      authorLink:
+        "http://lassewesth.blogspot.com/2018/07/the-international-e-road-network-and.html",
+      description: `A dataset containing European Roads.`,
+      queries: [
+        `CREATE CONSTRAINT ON (p:Place) ASSERT p.name IS UNIQUE`,
```

Comment:
is this still correct syntax?

Reply:
similar as the other, this is 4.x syntax, while 5.x is https://github.com/neo4j-devtools/neuler/blob/0d83eb767ba81640ff357246a4905b6e733153f6/src/components/SampleGraphs/sampleGraphs.js#L251